### DISCUSSION TITLE
I was wondering if it is possible to change the order views

### DISCUSSION THREAD

**Discussion by Bstijn** [community/community#7310] [Projects and Issues] [answered]
Right now you can't move the order of views. 

This is currently how my views are:
![image](https://user-images.githubusercontent.com/25815021/140403574-fca722c1-cccc-4c46-b793-3691c8372f19.png)

I Would like to change the order so that the focus to backlog view will be one position to the left.
![image](https://user-images.githubusercontent.com/25815021/140403753-aca7f5da-07a9-49d7-9c7c-91754b95f583.png)

Is there any way to do that as of this moment?

**Comment by jaredsb** [comment, marked as answer]
Not yet, but that should be available soon!
